src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -3722,15 +3724,15 @@ s32 func_80838144(s32 arg0) {
     }
 }
 
-s32 func_8083816C(s32 arg0) {
-    return (arg0 == FLOOR_TYPE_4) || (arg0 == FLOOR_TYPE_7) || (arg0 == FLOOR_TYPE_12);
+s32 func_8083816C(s32 floorType) {
+    return (floorType == FLOOR_TYPE_4) || (floorType == FLOOR_TYPE_7) || (floorType == FLOOR_TYPE_12);
 }
 
-void func_8083819C(Player* this, PlayState* play) {
+void Player_TryBurnShield(Player* this, PlayState* play) {
```

Comment:
would say deku shield in the name. may be obvious to most that deku is the only one that can burn, but yea